README.md

```diff
@@ -92,8 +92,6 @@ Right now we are working on the client part.
 
 tox-rs has contributions from many users. See [AUTHORS.md](/AUTHORS.md). Thanks everyone!
 
-tox-rs is sponsored by [Zerho LLC](https://zerho.info/).
```

Comment:
was sponsored?*

Reply:
Let it go.

Reply:
See our LICENSE-MIT

Reply:
Now I tweaked it a bit. I'm not sure if removing Zerho from the license is safe, so I kept it.